src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx

```diff
@@ -19,7 +19,13 @@ const getFromToNums = (page, total) => {
   return [from, to];
 };
 
-const Payments = ({ data, fields, tableVersion, textContent }) => {
+const Payments = ({
+  data,
+  fields,
+  tableVersion,
+  textContent,
+  alertMessage,
```

Comment:
Make sure you add `alertMessage` to the propTypes for the component

Reply:
Good shout - it has been a while since I have worked on a project that used `propTypes` 🙂